Makefile

```diff
@@ -51,6 +51,7 @@ rebuild-cluster: delete-cluster
 	./scripts/deploy-k8s-cluster.sh
 	./scripts/deploy-calico.sh
 	./scripts/preload-images.sh
+	./scripts/delete-standard-storageclass.sh
```

Comment:
Is it necessary to keep it under `install` as well?

Reply:
I kept it there as a "safeguard" but I guess yeah it could be removed from there.  I'll adjust.

Reply:
Actually yeah I'm going to keep it there as the safeguard.  It has an --ignore-not-found flag so it'll be a no-op.